lib/components/axe-violations.jsx

```diff
@@ -0,0 +1,62 @@
+// @flow
+
+const React = require('react');
+
+const { cmpStr } = require('../util');
+
+/*::
+import type {Record} from './table';
+
+type Props = {
+  records: Array<Record>;
+};
+*/
+
+function AxeViolations(props /*: Props */) {
+  const violationsMap /*: { [string]: number } */ = {};
+  const helpUrlsMap /*: { [string]: string } */ = {};
+
+  props.records.forEach(p => {
+    p.axeStats.violations.forEach(v => {
+      if (!(v.kind in violationsMap)) {
+        violationsMap[v.kind] = 0;
+        helpUrlsMap[v.kind] = v.helpUrl;
+      }
+      violationsMap[v.kind]++;
+    });
+  });
+
+  const violations = Object.keys(violationsMap).map(key => ({
+    name: key,
+    helpUrl: helpUrlsMap[key],
+    count: violationsMap[key]
+  }));
+
+  violations.sort((a, b) => {
+    if (b.count === a.count) {
+      return cmpStr(a.name, b.name);
+    }
+    return b.count - a.count;
+  });
```

Comment:
Hmm, this code should really be moved into a separate function so we can write unit tests for it.